### DISCUSSION TITLE
Error code when Writing SAVE file to SRAM W24258S (256K)

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#945] [General Chat]
Hello Sanny,

I get error some code when I try to write save file to SRAM.
Can you help me please to resolve the problem.
Thank's
---------
Cart Reader V3
Software version 6.4, updated to 7.3
---------
SRAM: W24258S (256K)
Game: LoROM
PCB: Reproduction
![IMG_6469](https://github.com/sanni/cartreader/assets/38383423/fc7cc554-9cb2-42be-a107-7f423e5e89c4)

**SRAM Writing Finished
ERROR: 16367 bytes
did not verify.**

When I do "TEST SRAM" I have also an error code
**0x0: ERROR
0XFF: OK**

I tried to see saved file witch was done with "TEST SRAM" function :
_Saved to SNES/SAVE/LuxiferAngel/29/LuxiferAngel.srm_
File was empty inside.

![IMG_6467](https://github.com/sanni/cartreader/assets/38383423/d66a3fe5-2df4-4bd8-8ad3-1ce8a0a65963)
![IMG_6468](https://github.com/sanni/cartreader/assets/38383423/5b75ee30-7389-418a-8811-c2dd4ead34a4)

After updated software to 7.3
Reading SAVE function was successful, with no error, but when writing I still get error.
![IMG_6474](https://github.com/sanni/cartreader/assets/38383423/a8a89171-5015-40a6-9e4f-7cd667a1f208)
![IMG_6475](https://github.com/sanni/cartreader/assets/38383423/f5153466-5afc-4fed-b14f-78666e9ebf07)

PS: 
I made my save file of game progress on my computer then I transfer saved ".srm" file to SD card,
Game is: Fire Emblem (LoROM, 32Mbit, SRAM 256K)

Any way, after that SAVE writing (with error code), I tested the game just to see, and there was no SAVE progress found. 

I tried to do Writing SRAM test with HiROM game (HiROM reproduction PCB )and W2465S-70LL SRAM (64K size).
When reading or writing  there is no error but when file was written to SRAM after my SNES test or Computer test (with saved file), there is no progression saved.
I checked saved file with TEXTEDIT and it's not empty.
![IMG_6470](https://github.com/sanni/cartreader/assets/38383423/228998a0-f3ae-438c-a7f1-0fde1f2de38e)

When I playing and doing saves of that HiROM PCB on my SNES, progress game file saved and loaded successfully.

What's wrong ?

**Comment by Chomemel** [comment]
Maybe it's also related to my issue.
[https://github.com/sanni/cartreader/discussions/936](url)
When I dump a working save from a Lorom/exLorom repro, the save won't work with the same rom on an emulator. I haven't try to write a save from the computer to the repro, but I guess I'd get the thing you do.
So far for the Hirom/exHirom repro playing a save dumped from the repro works fine on an emulator.
